### ISSUE TITLE
ClearML shows no scalars

### ISSUE THREAD

**Issue by bzisl** [Deci-AI/super-gradients#1145]
### 💡 Your Question

I'm using next parameters inside the train parameters:
```
                       normalize_targets=True,
               "metric_to_watch": 'mAP@0.50',
               "sg_logger": "clearml_sg_logger", # ClearML Logger, see class ClearMLSGLogger for details
               "sg_logger_params": { # parameters that will be passes to __init__ of the logger 
                   "project_name": project, # ClearML project name
                   "save_checkpoints_remote": True,
                   "save_tensorboard_remote": True,
                   "save_logs_remote": True,
                   } 
```
But no scalars are being show during the training, only an empty white screen is shown

### Versions

Collecting environment information...
PyTorch version: 2.0.1+cu117
Is debug build: False
CUDA used to build PyTorch: 11.7
ROCM used to build PyTorch: N/A

OS: Ubuntu 22.04.2 LTS (x86_64)
GCC version: (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0
Clang version: Could not collect
CMake version: version 3.26.3
Libc version: glibc-2.35

Python version: 3.9.16 (main, Dec  7 2022, 01:12:08)  [GCC 11.3.0] (64-bit runtime)
Python platform: Linux-5.15.0-71-generic-x86_64-with-glibc2.35
Is CUDA available: True
CUDA runtime version: 11.3.109
CUDA_MODULE_LOADING set to: LAZY
GPU models and configuration: 
GPU 0: NVIDIA GeForce GT 1030
GPU 1: NVIDIA GeForce RTX 3090

Versions of relevant libraries:
[pip3] flake8==5.0.4
[pip3] mypy-extensions==1.0.0
[pip3] numpy==1.23.0
[pip3] torch==2.0.1
[pip3] torchmetrics==0.8.0
[pip3] torchvision==0.15.2
[pip3] triton==2.0.0
[conda] Could not collect
